examples/Dashboard/webpack.config.js

```diff
@@ -1,19 +1,22 @@
 const ModuleFederationPlugin = require('webpack/lib/container/ModuleFederationPlugin');
-const LimitChunkCountPlugin = require('webpack/lib/optimize/LimitChunkCountPlugin');
-const CopyPlugin = require('copy-webpack-plugin');
 const TsconfigPathsPlugin = require('tsconfig-paths-webpack-plugin');
 
 const prod = process.env.NODE_ENV === 'production';
-const { dependencies } = require('./package.json');
+
 const path = require('path');
+const { name } = require('./package.json');
```

Comment:
Ignore this one, using webpack config from core: https://github.com/msupply-foundation/open-msupply/blob/9d726f1d9e2fd90f717e7fb763a24ac6980e9cb6/client/packages/plugins/plugin-webpack.config.js, and in plugins: https://github.com/msupply-foundation/open-msupply-plugins/blob/d9f0bd47a0a2f11ff1a6da934646afb9daf47aad/frontend/latest/webpack.config.js (this is in downstream PR)